src/main/resources/db/migration/V004__Add_events_table.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE events (
+  id          BIGSERIAL     PRIMARY KEY,
+  container   VARCHAR(50)   NOT NULL,
+  file_name   VARCHAR(255)  NOT NULL,
+  created_at  TIMESTAMP     NOT NULL,
+  event       VARCHAR(100)  NOT NULL,
+  notes       TEXT          NULL
+);
+
+CREATE INDEX events_container_filename_idx ON events (container, file_name);
+CREATE INDEX events_container_event_idx ON events (container, event);
```

Comment:
Seems a bit premature.
Why the second index?

Reply:
Can remove as no need at the given state